README.md

```diff
@@ -21,4 +21,6 @@ https://secret-wave-52844.herokuapp.com/
 ### Tests
 
 * Run unit tests `gradle test`
-* Run integration tests from root `cd gisrc/integration-test/http_server_spec/ && bundle exec spinach`
+* To run integration tests
+  - Start server locally `gradle run -Ddirectory=/src/integration-test/http_server_spec/data`. This will pull data from test file. 
```

Comment:
Thank you 🙇 A couple lines like this helps a lot, and usually just takes a few minutes to write.  A little documentation goes a long way.